java-test-samples/fargate-lambda-stepfunction-async/integration-test/lambda_function_cloudwatch.py

```diff
@@ -0,0 +1,73 @@
+import os
```

Comment:
What was the rationale to use a Python-based instead of a Java-based Lambda function?

Reply:
No rational behind in choosing Python. Same can be done by Java, RestAssured, Cucumber combination as well. Since there is no direct dependency on language of an application for e2e testing, We considered application development & testing as 2 different units of work.

Reply:
We chose Python due to its alignment with our team's expertise and ease of use, leading to faster testing cycles. Test framework is independent of the main architecture and in microservices based architecture, we can use any programming language, there is no specific reason of not choosing lambda using Java. 
In our decision-making process, we also carefully assessed factors such as Cold Start Performance, Resource Consumption, Code Readability, and Conciseness, where Python emerged as the optimal choice.